docs/modules/docker.md

```diff
@@ -21,7 +21,7 @@ docker:
 
 ## Screenshots
 
-<img class="screenshot" src="/assets/modules/docker.png" width="275" height="320" alt="github screenshot" />
+<img class="screenshot" src="/assets/modules/docker.png" width="275" height="320" alt="docker screenshot" />
```

Comment:
I would have made this change in a separate PR. It could have gotten merged much sooner that way.